module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,9 @@
+#include "vowel.hpp"
+
+void removeVowels(std::vector<std::string>& strings){
+    const std::array<char,17> vowels ={'a','e','i','o','u','y','A','E','I','O','U','Y'};
```

Comment:
Firstly you have constexpr int vowels_number=17 in the header,so why you use here magic '17' instead of that variable.
Secondly I think that this array should be constrexpr in the header as well.

Reply:
You can make it constexpr. And also named 17 value.

Reply:
Thanks, mistakes because of rush